android/app/google-services.json

```diff
@@ -1,40 +1,72 @@
 {
   "project_info": {
-    "project_number": "347734179578",
-    "project_id": "app-2i2i",
-    "storage_bucket": "app-2i2i.appspot.com"
+    "project_number": "453884442411",
+    "project_id": "i2i-test",
+    "storage_bucket": "i2i-test.appspot.com"
```

Comment:
why test? for the prod app, we should not have anything with test

Reply:
This is just latest json from firebase.

Reply:
This has been fixed.